Workaround build failure:

src/Language/Fixpoint/Types/PrettyPrint.hs:134:10: error:
    • Overlapping instances for PPrint String
        arising from a use of ‘Language.Fixpoint.Types.PrettyPrint.$dmpprintPrec’
      Matching instances:
        instance PPrint a => PPrint [a]
          -- Defined at src/Language/Fixpoint/Types/PrettyPrint.hs:102:10
        instance PPrint String
          -- Defined at src/Language/Fixpoint/Types/PrettyPrint.hs:134:10
    • In the expression:
        Language.Fixpoint.Types.PrettyPrint.$dmpprintPrec @String
      In an equation for ‘pprintPrec’:
          pprintPrec
            = Language.Fixpoint.Types.PrettyPrint.$dmpprintPrec @String
      In the instance declaration for ‘PPrint String’
diff --git a/src/Language/Fixpoint/Types/PrettyPrint.hs b/src/Language/Fixpoint/Types/PrettyPrint.hs
index 646ecaa..55c5b87 100644
--- a/src/Language/Fixpoint/Types/PrettyPrint.hs
+++ b/src/Language/Fixpoint/Types/PrettyPrint.hs
@@ -3,2 +3,3 @@
 {-# LANGUAGE OverloadedStrings  #-}
+{-# LANGUAGE OverlappingInstances  #-}
 
